asdf_astropy/converters/fits/fitswcs.py

```diff
@@ -0,0 +1,17 @@
+from asdf.extension import Converter
+
+
+class FitsWCSConverter(Converter):
+    tags = ("tag:astropy.org:astropy/fits/fitswcs-*",)
+    types = ("astropy.wcs.wcs.WCS",)
+
+    def from_yaml_tree(self, node, tag, ctx):
+        from astropy.wcs import WCS
+
+        primary_hdu = node["hdu"][0]
```

Comment:
```suggestion
        primary_hdu = node["hdu"][0]
```

Doesn't this throw out the distortion (and other non-primary) data that is generated with `to_fits`? Would you add a test with a wcs that generates more than 1 hdu for `to_fits`?

Reply:
@braingram, Do you have any suggestions for effectively preserving distortion data?

Reply:
I am currently experimenting and trying to determine the appropriate serialization logic for this [WCS](https://github.com/astropy/astropy/blob/d31407ddb99904263448d13bf87659b79d1cc0cc/astropy/wcs/tests/test_wcs.py#L1875C1-L1875C34). If you have any suggestions, please let me know.